ci/Dockerfile.aarch64

```diff
@@ -0,0 +1,55 @@
+ARG CUDA_VERSION=11.5.2
+FROM --platform=arm64 nvidia/cuda:$CUDA_VERSION-devel-ubuntu18.04
+
+RUN apt update && \
+apt-get -y install software-properties-common && \
+add-apt-repository ppa:ubuntu-toolchain-r/test && \
+apt update && \
+apt-get -y install gcc-9 g++-9 && \
```

Comment:
Has it been verified that the resulting binaries from the newer toolchain can still run properly on a vanilla Ubuntu 18.04 system?  Often installing a new toolchain means a newer C or C++ runtime library is required by the binaries produced by the newer toolchain.

Reply:
You are reasonable, the jar built on this docker image failed to run on a vanila ubuntu18 system. 
on the AWS ubuntu18 instance:
```
ubuntu@ip-172-31-62-87:~/plugin/aarch64/Linux$ ldd libcudf.so
./libcudf.so: /usr/lib/aarch64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by ./libcudf.so)
```
because it support until 3.4.25
```
ubuntu@ip-172-31-62-87:~/plugin/aarch64/Linux$ strings /usr/lib/aarch64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_3.4.17
GLIBCXX_3.4.18
GLIBCXX_3.4.19
GLIBCXX_3.4.20
GLIBCXX_3.4.21
GLIBCXX_3.4.22
GLIBCXX_3.4.23
GLIBCXX_3.4.24
GLIBCXX_3.4.25
GLIBCXX_DEBUG_MESSAGE_LENGTH
```

Let me try the rockylinux8 with devtoolset build.